it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersSoleProprietorUtrControllerISpec.scala

```diff
@@ -5,64 +5,64 @@ import play.api.libs.ws.DefaultWSCookie
 import play.api.test.WsTestClient
 import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
 
-class BusinessPartnersCorporateBodyUtrControllerIntegrationSpec
+class BusinessPartnersSoleProprietorUtrControllerISpec
   extends Specifications with TestConfiguration {
 
-  "GET /form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference" should {
+  val route = "/business-partners/partner-self-assessment-unique-taxpayer-reference"
+
+  s"GET /business-partners/partner-self-assessment-unique-taxpayer-reference" should {
 
     "render the partnership-self-assessment-unique-taxpayer-reference page" in {
       given
         .commonPrecondition
 
       WsTestClient.withClient { client =>
-        val result = client.url(s"$baseUrl/form/business-partners/partnership-self-assessment-unique-taxpayer-reference")
+        val result = client.url(s"$baseUrl" + s"$route")
```

Comment:
See string comment above - from comments on it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersSoleProprietorUtrControllerISpec.scala